go.disallow

```diff
@@ -0,0 +1,4 @@
+github.com/test-go/testify/assert Use github.com/stretchr/testify/assert instead
+github.com/test-go/testify/mock Use github.com/stretchr/testify/mock instead
+github.com/test-go/testify/require Use github.com/stretchr/testify/require instead
+go.uber.org/multierr Use the standard library instead, for example https://pkg.go.dev/errors#Join
```

Comment:
Maybe we could add our old fork of sqlx too?

Reply:
Also google's UUID lib

Reply:
```
github.com/smartcontractkit/sqlx Use github.com/jmoiron/sqlx instead
```
Which uuid module? We prefer `github.com/google/uuid` now. I don't recall the ones we removed :thinking:

Reply:
I guess there is `github.com/gofrs/uuid` coming indirectly from `jackc/pgx`.

Reply:
Oh also `github.com/satori/go.uuid`, so:
```
github.com/gofrs/uuid Use github.com/google/uuid instead
github.com/satori/go.uuid Use github.com/google/uuid instead
```

Reply:
What is our preferred `null` module? `gopkg.in/guregu/null.v4`? (I see one stray v2 usage)

Reply:
Updated.